src/botPage/view/View.js

```diff
@@ -455,6 +455,7 @@ export default class View {
         };
 
         const showSummary = () => {
+            $('#summaryPanel').dialog('option', 'minWidth', 770);
```

Comment:
You can chain the commands here so we don't have to select the same element in the DOM twice, accessing the DOM is expensive, so try to do it as little as possible. 

Example:
```JavaScript
$('#summaryPanel')
    .dialog('option', 'minWidth', 770)
    .dialog('open');
```